modules/tinymce/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed a workaround for ensuring stylesheets are loaded in an outdated version of webkit. #TINY-9433
 - Lists in a noneditable root were incorrectly editable using list API commands, toolbar buttons and menu items. #TINY-9458
 - Color picker dialog would not update the preview color if the hex input value was prefixed with `#` symbol. #TINY-9457
+- Fix determining if the current selection contains a link and retrieving its href value. #TINY-7993
```

Comment:
As you can see in the entries for the Fixed section we avoid using redundant 'Fix' word. Instead, we explain what the bug was.
```suggestion
- The Edit Link dialog incorrectly retrieved the URL value when opened immediately after the link insertion. #TINY-7993
```

Reply:
Thanks for suggesting this! Fixed [here](https://github.com/tinymce/tinymce/pull/8386/commits/5ffe360f5139ff1f7e80faaa952457b4af5b8660).